Successfully updated:
Created Frame node enVjFnA6R
Created Frame node ukT6MY5Rp
Created Text node Y45ae97Ow
Created Text node CWzR3GBpE
Created Frame node AuVIlqLiO
Updated attributes for node enVjFnA6R
Updated attributes for node ukT6MY5Rp
Updated text for node Y45ae97Ow
Updated attributes for node Y45ae97Ow
Updated text for node CWzR3GBpE
Updated attributes for node CWzR3GBpE
Updated attributes for node AuVIlqLiO
Reordered node ukT6MY5Rp within parent enVjFnA6R to index 0

XML Changes:
Index: node.xml
===================================================================
--- node.xml	Before
+++ node.xml	After
@@ -3611,20 +3611,73 @@
       Header Text
     </HeaderText>
   </Frame>
   <BodyContentGoesHere
       nodeId="LXzPmG6zb"
       width="fit-content"
       height="fit-content"
       font="Inter"
   >
     Body content goes here
   </BodyContentGoesHere>
   <Frame
       nodeId="xLL2w0cI7"
       width="100%"
       height="1fr"
       <!-- background color string or project color style path (if starts with /) -->
       backgroundColor="rgb(255, 255, 255)"
       borderRadius="4px"
    />
 </Stack>
+<Stack
+    nodeId="enVjFnA6R"
+    position="absolute"
+    width="400px"
+    height="300px"
+    top="0px"
+    left="0px"
+    <!-- background color string or project color style path (if starts with /) -->
+    backgroundColor="rgb(240, 240, 240)"
+    layout="stack"
+    gap="16px"
+    padding="20px"
+    stackDirection="vertical"
+    stackDistribution="center"
+    stackAlignment="center"
+>
+  <Frame
+      nodeId="ukT6MY5Rp"
+      width="100%"
+      height="60px"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(100, 150, 200)"
+      borderRadius="8px"
+  >
+    <HeaderText
+        nodeId="Y45ae97Ow"
+        position="absolute"
+        width="fit-content"
+        height="fit-content"
+        top="0px"
+        left="0px"
+        font="Inter"
+    >
+      Header Text
+    </HeaderText>
+  </Frame>
+  <BodyContentGoesHere
+      nodeId="CWzR3GBpE"
+      width="fit-content"
+      height="fit-content"
+      font="Inter"
+  >
+    Body content goes here
+  </BodyContentGoesHere>
+  <Frame
+      nodeId="AuVIlqLiO"
+      width="100%"
+      height="1fr"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(255, 255, 255)"
+      borderRadius="4px"
+   />
+</Stack>


Note: Set zoomIntoView=false if you want to use Framer app while MCP is working.